### DISCUSSION TITLE
CORS issue

### DISCUSSION THREAD

**Discussion by palsypentertainment** [orestbida/cookieconsent#735] [Q&A]
Love the script. I am currently setting up CORS policies (in report-only mode for now) and have encountered a minor issue or annoyance.  Although everything still works (obviously, in reporting-mode), am seeing this:

![cors](https://github.com/user-attachments/assets/bf0e961a-57d1-46e8-93be-25f864ad7172)

The main script is _imported_ in the config.js, not sure if that has anything to do with it.

Thanks.

**Comment by orestbida** [comment]
This doesn't have anything to do with the plugin but rather with the CSP configuration. It says that inline scripts are not allowed, so I assume you are using inline scripts.

**Reply by palsypentertainment** [reply]
I spent the day moving all my JS to external files with an integrity of sha384 and a crossorigin....... So am not sure how it's flagging that I am.

**Reply by orestbida** [reply]
Weird, I would need to check the site if there'a live url. Did you try moving the "import" statement outside the config. script?

**Reply by palsypentertainment** [reply]
Still having issues with this....... I need to add _'unsafe-inline'_ to the _script-src_ to make CORS happy.
>Did you try moving the "import" statement outside the config. script?

Can you explain this. Do you mean just include the _*.umd.js_ in a <script> tag?
